tracer/src/Datadog.Trace/Debugger/Upload/SymbolUploadApi.cs

```diff
@@ -90,11 +90,13 @@ public override async Task<bool> SendBatchAsync(ArraySegment<byte> symbols)
 #if NETFRAMEWORK
                 using var gzipStream = new Vendors.ICSharpCode.SharpZipLib.GZip.GZipOutputStream(memoryStream);
                 await gzipStream.WriteAsync(symbols.Array, 0, symbols.Array.Length).ConfigureAwait(false);
+                await gzipStream.FlushAsync().ConfigureAwait(false);
 #else
                 using var gzipStream = new GZipStream(memoryStream, CompressionMode.Compress);
                 await gzipStream.WriteAsync(symbols.Array, 0, symbols.Array.Length).ConfigureAwait(false);
+                await gzipStream.FlushAsync().ConfigureAwait(false);
```

Comment:
nit: since those 2 lines are the same code, albeit on a different type, they could be outside the #if/else